packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -174,8 +169,8 @@ export const DefaultConfiguration: Configuration = {
     },
   },
   screenShare: {
-    audio: true,
-    video: true,
+    audio: false, // todo: SPARK-393485
+    video: false, // todo: SPARK-393485
```

Comment:
Why have this specifically instead of inferring it from the layouts?

Reply:
I'm just here to sow confusion

Reply:
I've done this change. Now we have `numOfScreenShareStreams` in the audio part of the `Configuration` and I've removed the `screenShare` section from it completely. Whether we create the screen share video receive slot or not is now inferred from the layouts.